smart_campus/app/models.py

```diff
@@ -1,3 +1,222 @@
 from django.contrib.gis.db import models
+from django.contrib.auth.base_user import (
+    BaseUserManager, AbstractBaseUser
+)
 
-# Create your models here.
+class Beacon(models.Model):
```

Comment:
Add one more blank line above.
According to [PEP8 - Blank lines], you should surround top-level function and class definitions with two blank lines.